Sharpasonne.Models/Placement.cs

```diff
@@ -1,22 +1,28 @@
-using System;
+using JetBrains.Annotations;
+using Optional;
 
 namespace Sharpasonne.Models
 {
     public class Placement
     {
-        public Tile Tile { get; }
-        public Orientation Orientation { get; }
+        [NotNull] public TilePlacement           TilePlacement   { get; }
```

Comment:
no horiz allinment, it's going to fuck up git history.

Reply:
Oh I'll never stop.